makeflow/src/dag_file.c

```diff
@@ -84,4 +87,32 @@ int dag_file_in_trans( const struct dag_file *f )
 		return 0;
 }
 
+int dag_file_mount_clean(struct dag *d) {
```

Comment:
The structure of this function/location aren't very clear. This function should operate on a single dag_file and clean the cache information. If you want to do an operation on the list of dag_files that should be in dag.c.